Templates/CDS-CSharp/run.csx

```diff
@@ -0,0 +1,67 @@
+﻿#load ".\SecurityHelper.csx"
+#load ".\TraceWriterTelemetryBridge.csx"
+
+using Microsoft.CommonDataService;
+using Microsoft.CommonDataService.CommonEntitySets;
+using Microsoft.CommonDataService.Configuration;
+using Microsoft.CommonDataService.ServiceClient.Security;
+using Microsoft.CommonDataService.ServiceClient.Security.Credentials;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+class EnvironmentValues
```

Comment:
Agreed these will eventually be app settings driven by Azure function UI to setup EasyAuth and Environment selection. Consensus with Matthew was for the first iteration to ship with these hardcoded.